### ISSUE TITLE
Table overflows the body.

### ISSUE THREAD

**Issue by oluwakayode-a** [themesberg/flowbite#111]
**Describe the bug**
I noticed that the overflow-hidden isn't applied to the table component, and it overflows the body on mobile view. I observed the same on the website as well.

**To Reproduce**
1. Add Table element
2. Toggle mobile view

**Expected behavior**
The table should not overflow the body.

**Screenshots**
![image](https://user-images.githubusercontent.com/32282156/158345584-9609f096-3a1d-468c-9d6b-180e94f93d08.png)

**Comment by zoltanszogyenyi** [comment]
Hey @oluwakayode-a,

Fixed this problem for the v1.4.0 release.

Here's an example of a table element without the overflow issue:

```
<div class="relative overflow-x-auto shadow-md sm:rounded-lg">
    <table class="w-full text-sm text-left text-gray-500 dark:text-gray-400">
        <thead class="text-xs text-gray-700 uppercase bg-gray-50 dark:bg-gray-700 dark:text-gray-400">
            <tr>
                <th scope="col" class="px-6 py-3">
                    Product name
                </th>
                <th scope="col" class="px-6 py-3">
                    Color
                </th>
                <th scope="col" class="px-6 py-3">
                    Category
                </th>
                <th scope="col" class="px-6 py-3">
                    Price
                </th>
                <th scope="col" class="px-6 py-3">
                    <span class="sr-only">Edit</span>
                </th>
            </tr>
        </thead>
        <tbody>
            <tr class="bg-white border-b dark:bg-gray-800 dark:border-gray-700">
                <th scope="row" class="px-6 py-4 font-medium text-gray-900 dark:text-white whitespace-nowrap">
                    Apple MacBook Pro 17"
                </th>
                <td class="px-6 py-4">
                    Sliver
                </td>
                <td class="px-6 py-4">
                    Laptop
                </td>
                <td class="px-6 py-4">
                    $2999
                </td>
                <td class="px-6 py-4 text-right">
                    <a href="#" class="font-medium text-blue-600 dark:text-blue-500 hover:underline">Edit</a>
                </td>
            </tr>
            <tr class="bg-white border-b dark:bg-gray-800 dark:border-gray-700">
                <th scope="row" class="px-6 py-4 font-medium text-gray-900 dark:text-white whitespace-nowrap">
                    Microsoft Surface Pro
                </th>
                <td class="px-6 py-4">
                    White
                </td>
                <td class="px-6 py-4">
                    Laptop PC
                </td>
                <td class="px-6 py-4">
                    $1999
                </td>
                <td class="px-6 py-4 text-right">
                    <a href="#" class="font-medium text-blue-600 dark:text-blue-500 hover:underline">Edit</a>
                </td>
            </tr>
            <tr class="bg-white dark:bg-gray-800">
                <th scope="row" class="px-6 py-4 font-medium text-gray-900 dark:text-white whitespace-nowrap">
                    Magic Mouse 2
                </th>
                <td class="px-6 py-4">
                    Black
                </td>
                <td class="px-6 py-4">
                    Accessories
                </td>
                <td class="px-6 py-4">
                    $99
                </td>
                <td class="px-6 py-4 text-right">
                    <a href="#" class="font-medium text-blue-600 dark:text-blue-500 hover:underline">Edit</a>
                </td>
            </tr>
        </tbody>
    </table>
</div>
```

**Comment by oluwakayode-a** [comment]
> Hey @oluwakayode-a,
> 
> Fixed this problem for the v1.4.0 release.
> 
> Here's an example of a table element without the overflow issue:
> 
> ```
> <div class="relative overflow-x-auto shadow-md sm:rounded-lg">
>     <table class="w-full text-sm text-left text-gray-500 dark:text-gray-400">
>         <thead class="text-xs text-gray-700 uppercase bg-gray-50 dark:bg-gray-700 dark:text-gray-400">
>             <tr>
>                 <th scope="col" class="px-6 py-3">
>                     Product name
>                 </th>
>                 <th scope="col" class="px-6 py-3">
>                     Color
>                 </th>
>                 <th scope="col" class="px-6 py-3">
>                     Category
>                 </th>
>                 <th scope="col" class="px-6 py-3">
>                     Price
>                 </th>
>                 <th scope="col" class="px-6 py-3">
>                     <span class="sr-only">Edit</span>
>                 </th>
>             </tr>
>         </thead>
>         <tbody>
>             <tr class="bg-white border-b dark:bg-gray-800 dark:border-gray-700">
>                 <th scope="row" class="px-6 py-4 font-medium text-gray-900 dark:text-white whitespace-nowrap">
>                     Apple MacBook Pro 17"
>                 </th>
>                 <td class="px-6 py-4">
>                     Sliver
>                 </td>
>                 <td class="px-6 py-4">
>                     Laptop
>                 </td>
>                 <td class="px-6 py-4">
>                     $2999
>                 </td>
>                 <td class="px-6 py-4 text-right">
>                     <a href="#" class="font-medium text-blue-600 dark:text-blue-500 hover:underline">Edit</a>
>                 </td>
>             </tr>
>             <tr class="bg-white border-b dark:bg-gray-800 dark:border-gray-700">
>                 <th scope="row" class="px-6 py-4 font-medium text-gray-900 dark:text-white whitespace-nowrap">
>                     Microsoft Surface Pro
>                 </th>
>                 <td class="px-6 py-4">
>                     White
>                 </td>
>                 <td class="px-6 py-4">
>                     Laptop PC
>                 </td>
>                 <td class="px-6 py-4">
>                     $1999
>                 </td>
>                 <td class="px-6 py-4 text-right">
>                     <a href="#" class="font-medium text-blue-600 dark:text-blue-500 hover:underline">Edit</a>
>                 </td>
>             </tr>
>             <tr class="bg-white dark:bg-gray-800">
>                 <th scope="row" class="px-6 py-4 font-medium text-gray-900 dark:text-white whitespace-nowrap">
>                     Magic Mouse 2
>                 </th>
>                 <td class="px-6 py-4">
>                     Black
>                 </td>
>                 <td class="px-6 py-4">
>                     Accessories
>                 </td>
>                 <td class="px-6 py-4">
>                     $99
>                 </td>
>                 <td class="px-6 py-4 text-right">
>                     <a href="#" class="font-medium text-blue-600 dark:text-blue-500 hover:underline">Edit</a>
>                 </td>
>             </tr>
>         </tbody>
>     </table>
> </div>
> ```

Thank you!!!!!
